pifi/videoprocessor.py

```diff
@@ -469,6 +488,11 @@ def __do_housekeeping(self, clear_screen = True):
         cleanup_ffmpeg_to_python_fifos_cmd = f'sudo rm -rf {path_glob}'
         subprocess.check_output(cleanup_ffmpeg_to_python_fifos_cmd, shell = True, executable = '/usr/bin/bash')
 
+        self.__logger.info("Deleting fps fifos...")
+        path_glob = shlex.quote(tempfile.gettempdir() + "/" + 'fps') + '*'
+        cleanup_fps_fifos_cmd = f'sudo rm -rf {path_glob}'
+        subprocess.check_output(cleanup_fps_fifos_cmd, shell = True, executable = '/usr/bin/bash')
```

Comment:
if all the fifos we create share a common prefix, we can delete them with a single `rm` command. The logic for the common prefix could be in the `__make_fifo()` method I suggested.

Reply:
I guess you can also pass multiple paths to `rm` also, so no common prefix may be necessary. Up to you if you want to have a common prefix.